### PR TITLE
[feature][connector][cdc] CDC base classes

### PR DESCRIPTION
**Pull request by ashulin**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)

### CODE REVIEW

**Comment by ashulin** [comment]
@hailin0 @ic4y PTAL

File: seatunnel-connectors-v2/connector-cdc/connector-cdc-base/pom.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>connector-cdc</artifactId>
+        <groupId>org.apache.seatunnel</groupId>
+        <version>${revision}</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>connector-cdc-base</artifactId>
```
Comment (hailin0) [inline seatunnel-connectors-v2/connector-cdc/connector-cdc-base/pom.xml:29]: need to skip `maven-shade-plugin`?

Comment (ashulin) [reply seatunnel-connectors-v2/connector-cdc/connector-cdc-base/pom.xml:29]: done

**Review by ic4y — APPROVED** [review]
+1

**Review by hailin0 — APPROVED** [review]

**Review by EricJoy2048 — APPROVED** [review]

**Pull request merged**